org_status/org_status.py

```diff
@@ -75,10 +76,41 @@ def get_argument_parser():
     parser.add_argument('--verbose', '-v', action='store_true')
     parser.add_argument('--hosts-only', '-o', action='store_true')
     parser.add_argument('--skip-host-checks', action='store_true')
+    parser.add_argument('--export-orgs', type=str)
+    parser.add_argument('--format', type=str, default='gitman')
 
     return parser
 
 
+def encode_repo_list(repo_data, encoder_name, styled):
```

Comment:
Please don't have this method acess `styled` as an argument, this would make it tough for us to move over to a standard logger when we decide to do that. A better way would be to have this method return appropriate values or raise exceptions that should be caught and processed in `main()`. This method should not have these kind of side effects.

Reply:
Deferred to be part of https://github.com/ksdme/org-status/issues/23.